metrics/storage/webtooling.sh

```diff
@@ -100,26 +100,24 @@ function main() {
 		echo "Verify that the containers are running"
 		containers_launched=$(sudo ctr c list -q | wc -l)
 		[ "$containers_launched" -eq "$NUM_CONTAINERS" ] && break
- 		sleep 1
- 		[ "$i" == "$timeout" ] && return 1
+		sleep 1
+		[ "$i" == "$timeout" ] && return 1
 	done
 
 	# Launch webtooling benchmark
- 	CONTAINERS_ID=($(sudo ctr c list -q))
- 	for i in "${CONTAINERS_ID[@]}"; do
+	CONTAINERS_ID=($(sudo ctr c list -q))
+	for i in "${CONTAINERS_ID[@]}"; do
 		sudo ctr t exec -d --exec-id "$(random_name)" "$i" sh -c "$CMD"
- 	done
+	done
 
-	# Verify that webtooling benchmark finished
 	CONTAINERS_ID=($(sudo ctr c list -q))
-	for i in $(seq 1 "$timeout"); do
- 		FILE_CMD="cat $file_path/$file_name | grep Geometric"
- 		check_file_content=$(sudo ctr t exec --exec-id "$(random_name)" "$i" sh -c "$FILE_CMD")
+	for j in $(seq 1 "$timeout"); do
+		FILE_CMD="cat $file_path/$file_name | grep Geometric"
+		check_file_content=$(sudo ctr t exec --exec-id "$(random_name)" "$i" sh -c "$FILE_CMD")
```

Comment:
```suggestion
		check_file_content=$(sudo ctr t exec --exec-id "$(random_name)" "$j" sh -c "$FILE_CMD")
```
The suggestion is, as you changed to for to use j instead of i, you need to adapt that inside the loop.

Reply:
@fidencio if I use `j` instead of `i` it will take the variable of 1 to 180 which is something that I do not want, I am using i as it is the variable that is being used for the container id which is something that I need to use for the exec

Reply:
Oh, now I see what you're doing.  It's confusing for the reader, I must admit, mainly because you're not looping here using $i, but rather using the last container you exec'ed.

May I bother you to add a comment about that?  I'm pretty sure the next time I'm sleepy and look at the code I'll have the same questions.

And thanks for the explanation, @GabyCT.

Reply:
@fidencio your welcome and thanks for the feedback

Reply:
@fidencio changes applied